### PR TITLE
[TW-118] Refresh "Building request workflows"

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Rewrote and edited "Building request workflows" page
- Restructured content to be easier to scan
- Updated screenshots

### CODE REVIEW

File: src/pages/docs/running-collections/building-workflows.md [hunk 1]
```diff
@@ -26,42 +27,75 @@ warning: false
...
 
-[![set next request method](https://assets.postman.com/postman-docs/Test_script10.png)](https://assets.postman.com/postman-docs/Test_script10.png)
+As the name suggests, `postman.setNextRequest()` enables you to specify which request Postman runs next, following the current request. Using this function, you can build custom workflows that chain requests, running them one after the other in a specific order.
 
-## Set the request to be executed next
+<img alt="Setting the next request" src="https://assets.postman.com/postman-docs/set-next-request-v9-4.jpg" width="841px">
+
+## Setting the next request
+
+To specify the request to run next, add the following code on the **Tests** tab of a request. For `request_name`, use the name of the request you want to run next.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/building-workflows.md:40]: Maybe: 
```suggestion
To specify the request to run next, add the following code on the **Tests** tab of a request. Replace `request_name` with the name of the request you want to run next.
```

File: src/pages/docs/running-collections/building-workflows.md [hunk 2]
```diff
@@ -26,42 +27,75 @@ warning: false
...
 postman.setNextRequest("request_name");
 ```
 
-## Loop over the current request
+Postman runs the specified request after completing the current request.
+
+## Looping over a request
 
-Providing the name of current run to `setNextRequest` leads to Postman running the current request continuously.
+If you pass the name of the current request to the `setNextRequest` function, Postman will run the current request repeatedly on a continuous loop.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/building-workflows.md:50]: I think this sentence could omit either "repeatedly" or "continuous"

File: src/pages/docs/running-collections/building-workflows.md [hunk 3]
```diff
@@ -26,42 +27,75 @@ warning: false
...
+
+Keep the following tips in mind when using the `postman.setNextRequest()` function.
+
+### setNextRequest() only works in Collection Runner
+
+The `postman.setNextRequest()` function has no effect when you run a request using the **Send** button and is only used when you run a collection.
+
+### Use setNextRequest() in pre-request or test scripts
+
+You can use `postman.setNextRequest()` in the pre-request script or the test script of a request. If more than one valued is assigned, the last value that is set takes precedence.
```
Comment (HeatherStenson) [inline src/pages/docs/running-collections/building-workflows.md:76]:
```suggestion
You can use `postman.setNextRequest()` in the pre-request script or the test script of a request. If more than one value is assigned, the last value that is set takes precedence.
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**